app/models/participant_profile.rb

```diff
@@ -94,7 +94,7 @@ def validation_decision(name)
   end
 
   def state_at(declaration_date)
-    participant_profile_states.where("created_at < ?", declaration_date).order(:created_at).last
+    participant_profile_states.where("created_at <= ?", declaration_date).order(:created_at).last
```

Comment:
Is this unrelated to the PR subject? Please could you add or modify the spec for this?